app/src/main/java/de/tum/in/tumcampusapp/activities/PlansActivity.java

```diff
@@ -1,140 +1,27 @@
 package de.tum.in.tumcampusapp.activities;
-
-import android.content.DialogInterface;
-import android.content.Intent;
-import android.net.Uri;
 import android.os.Bundle;
-import android.support.v4.content.FileProvider;
-import android.support.v7.app.AlertDialog;
-import android.view.View;
-import android.widget.AdapterView;
-import android.widget.AdapterView.OnItemClickListener;
-import android.widget.ListView;
-import android.widget.ProgressBar;
-
-import com.google.common.collect.ImmutableList;
-
-import java.io.File;
-import java.io.IOException;
-import java.util.List;
+import android.support.v4.app.FragmentTransaction;
 
 import de.tum.in.tumcampusapp.R;
 import de.tum.in.tumcampusapp.activities.generic.BaseActivity;
-import de.tum.in.tumcampusapp.adapters.PlanListAdapter;
-import de.tum.in.tumcampusapp.adapters.PlanListAdapter.PlanListEntry;
-import de.tum.in.tumcampusapp.auxiliary.NetUtils;
-import de.tum.in.tumcampusapp.auxiliary.Utils;
+import de.tum.in.tumcampusapp.fragments.PlansViewFragment;
 
 /**
  * Activity to show plans.
  */
-public class PlansActivity extends BaseActivity implements OnItemClickListener {
-
-    private static final String[][] FILES_TO_DOWNLOAD = {
-            {"http://www.mvv-muenchen.de/fileadmin/media/Dateien/plaene/pdf/Netz_2016_Version_MVG.PDF", "Schnellbahnnetz.pdf"},
-            {"http://www.mvv-muenchen.de/fileadmin/media/Dateien/plaene/pdf/Nachtnetz_2016.pdf", "Nachtliniennetz.pdf"},
-            {"http://www.mvv-muenchen.de/fileadmin/media/Dateien/plaene/pdf/Tramnetz_2016.pdf", "Tramnetz.pdf"},
-            {"http://www.mvv-muenchen.de/fileadmin/media/Dateien/3_Tickets_Preise/dokumente/TARIFPLAN_2016-Innenraum.pdf", "Tarifplan.pdf"},
-    };
-    ProgressBar progressBar;
-    private final Thread downloadFiles = new Thread() {
```

Comment:
Since i currently disable the listview after the download finished and also set the progressbar as invisible, i thought it makes sense to put these things together. If you prefer the download in the activity itself, I can try to make it work!